<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RayleighSommerfeldForm</class>
 <widget class="QWidget" name="RayleighSommerfeldForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1436</width>
    <height>1587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>1250</y>
     <width>641</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="zmin_label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>zmin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>dz</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="zmax_label">
      <property name="text">
       <string>zmax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>zmax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>zmin</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="dz_label">
      <property name="text">
       <string>dz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="zmin_slider">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Back Propagate</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="zmax_slider">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="dz_slider">
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="pageStep">
       <number>20</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>1200</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="maxCount">
       <number>20</number>
      </property>
      <item>
       <property name="text">
        <string>Grayscale</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Coolwarm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Summer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viridis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Winter</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Choose input frame</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Viewing Options</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="openImage">
      <property name="text">
       <string>Browse Files</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Phase</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Colormap:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>Intensity</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>1281</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Input Frame</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>zmin_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zmin_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zmax_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zmax_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dz_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dz_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
